chain/indexer/src/lib.rs

```diff
@@ -92,7 +95,11 @@ pub struct Indexer {
 impl Indexer {
     /// Initialize Indexer by configuring `nearcore`
     pub fn new(indexer_config: IndexerConfig) -> Self {
+        tracing::info!(target: INDEXER, "Load config from {:?}...", &indexer_config.home_dir);
+
         let near_config = nearcore::load_config(&indexer_config.home_dir);
```

Comment:
`load_config` loads the whole nearcore "config" which is the `config.json` and `genesis.json` (don't even mention how the naming is misleading here). I would consider adding the logs to the `nearcore::load_config` function. Also, consider using `nearcore::load_config_without_genesis_records`.

Reply:
Why `nearcore::load_config_without_genesis_records` if we later use genesis records in indexers?

Reply:
OK, I'll consider but it will completely ruin the current way of handling genesis records on indexer for explorer side. I don't know if anyone else is storing genesis records on their indexers, so I assume it'll be a breaking change. Investigating at the moment will let you know.